### ISSUE TITLE
Item Idea, Hawk Summoning

### ISSUE THREAD

**Issue by prodigiousking** [AngelArenaAllstars/suggestions#106]
Build up =3 rings of aquila, gives you some solid stats as well as an aura idk numbers for balance that well so someone help me out plz. Active ability: Summons Hawk- summons hawk just like beastmaster so you have a mobile ward that you dont have to go into enemy territory to place, bird lasts certain duration, 2 charges. certain cooldown, certain bounty. I figured it would be a decent support item as it would give good stats, an aura for your team, make stacking aquilas more viable and interesting, and be an alternative to warding once again i have no clue as to what numbers would be balanced.

**Comment by prodigiousking** [comment]
Name is a work in progress, maybe Pendant or rod of aquila or something of the sort

**Comment by prodigiousking** [comment]
Might need a recipe or a different build up seems too powerful, also 2 charges might be a bit much just baed of of Angel arena reborns octarine core active being good and yet drastically less powerful. I also dont know if birds would go invis I like the idea of a powerful stats plus warding item for supports, vut I really need help making this item balanced.

**Comment by Haganeko** [comment]
Thanks for the post.
We're implementing Stackquila as an item already, as a tribute to... well, stackquila. Stealing spells from heroes is a bad idea, though - it makes the hero less special.

**Comment by prodigiousking** [comment]
Yeah i thought about that, but I mean its not as if anyone picks beastmaster for the bird, the ability automatically gets skilled and the ability wouldnt have to be exact it just implements the moveable ward mechanic, additionally blink dagger, for example, copies antimage and and queen of pain, but the numbers and the 3 second disable mechanic make it so qop and antimages blinks are still very powerful inbuilt. A possible nerf to the bird would be it can only perch or be invis on trees.

**Comment by Haganeko** [comment]
Those are interesting points.
So we're looking at an item that could be to beast master what blink dagger is to antimage - partial utility compared to the hero spell.
Perching on trees sounds like an interesting compromise - we'll see what others think of it.

**Comment by prodigiousking** [comment]
Cool
